.env.example

```
@@ -1,5 +1,4 @@
 ENCODER_KEY = password  # master key for encrypting data
-MAILJET_APIKEY_PUBLIC = d3522c8267bb0f41e84b35e51e7981618008a485  # mailjet mailer public API key
-MAILJET_APIKEY_PRIVATE = 636a6756861f8d411427ebf84d405445ad0d5f97  # mailjet mailer private API key
+SENDINBLUE_APIKEY_PUBLIC= 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 # SendInBlue mailer public API key
```

Why is this named “PUBLIC” when it is apparently private?

```suggestion
SENDINBLUE_API_KEY= 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 # SendInBlue mailer public API key
```